### PR TITLE
added highlight_fields property

### PR DESCRIPTION
**Pull request by agauch**
I added the property "highlight_fields" in the search parameters.

### CODE REVIEW

**Comment by runeanielsen** [comment]
Hi @agauch,

Thank you for the contribution, can I get you to add a bit more documentation to the docstring.

Example insert the docs from the Typesense API documentation:

"
Comma separated list of fields which should be highlighted fully without snippeting.

Default: all fields will be snippeted.

Set to none to disable highlighting fully.
"

// Rune

File: src/Typesense/SearchParameters.cs [hunk 1]
```diff
@@ -269,6 +269,12 @@ public record SearchParameters
     [JsonPropertyName("highlight_full_fields")]
     public string? HighlightFullFields { get; set; }
 
+    /// <summary>
+    /// List of fields which should be highlighted with snippetting.
+    /// </summary>
```
Comment (runeanielsen) [inline src/Typesense/SearchParameters.cs:274]: Add more documentation to the documentation, you could for example use the documentation from the API spec. https://typesense.org/docs/27.0/api/search.html#results-parameters

**Comment by agauch** [comment]
Hello @runeanielsen, 
The docstring is completed as requested.

**Comment by runeanielsen** [comment]
@agauch looks great, thank you very much for making the change. :)

// Rune

**Comment by runeanielsen** [comment]
Hi @agauch, I've made a new release with tag `7.16.0` with your changes.

**Comment by agauch** [comment]
Thanks!

**Pull request merged**